about.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>We Care</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet"
+    href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+</head>
+
+<body>
+  <section class="popup-nav">
+    <div class="popup-header">
+      <button class="cancel-button">
+        <img src="images/icons/Cancel-Icon.svg" alt="Cancel icon">
+      </button>
+    </div>
+    <ul class="popup-menu">
+      <li>
+        <a class="menu-item" href="index.html">
+          Home
+        </a>
+      </li>
+      <li>
+        <a class="menu-item" href="about.html">
+          About
+        </a>
+      </li>
+      <li>
+        <a class="menu-item2" href="#">
+          Program
+        </a>
+      </li>
+      <li>
+        <a class="menu-item3" href="#">
+          Join
+        </a>
+      </li>
+      <li>
+        <a class="menu-item4" href="#">
+          Sponser
+        </a>
+      </li>
+      <li>
+        <a class="menu-item5" href="#">
+          News
+        </a>
+      </li>
+    </ul>
+  </section>
+  <section class="about">
+    <header>
+      <div class="account">
+        <img src="images/icons/facebook-icon.svg" alt="facebook icon">
+        <img src="images/icons/twitter-icon.svg" alt="twitter icon">
+        <a href="#">English</a>
+        <a href="#">My Page</a>
+        <a href="#">Logout</a>
+      </div>
+      <nav>
+        <a href="#">
+          <i class="material-symbols-outlined menu">
+            menu  
+          </i>
+        </a>
+        <a href="index.html"><img src="images/logo_WECARE.svg" alt="logo"></a>
+        <div class="menu-items">
+          <ul>
+            <li><a href="#">About</a></li>
```

Comment:
- [OPTIONAL] It's better to change the `About` nav link to `Home` which will increase the User Experience(UX) on your website because we don't need to put a link to navigate to the current page we are on. It's better to add the nav link to navigate to the `Home` page instead.